src/Microsoft.ML.TimeSeries/SsaAnomalyDetectionBase.cs

```diff
@@ -186,13 +191,16 @@ public override void Save(ModelSaveContext ctx)
             // float: _discountFactor
             // byte: _errorFunction
             // bool: _isAdaptive
+            // State: StateRef
             // AdaptiveSingularSpectrumSequenceModeler: _model
 
             base.Save(ctx);
             ctx.Writer.Write(SeasonalWindowSize);
             ctx.Writer.Write(DiscountFactor);
             ctx.Writer.Write((byte)ErrorFunction);
             ctx.Writer.Write(IsAdaptive);
+            StateRef.Save(ctx);
```

Comment:
>StateRef.Save(ctx); [](start = 12, length = 19)

Actually this doesn't need to even be an abstract method... every state instance can just implement *a* method (internally, of course).